etc/ci/update-wpt-checkout

```diff
@@ -115,7 +115,11 @@ EOF
     curl -H "Authorization: token ${WPT_SYNC_TOKEN}" \
          -H "Content-Type: application/json" \
          --data @prdata.json \
-         https://api.github.com/repos/servo/servo/pulls || return 5
+         https://api.github.com/repos/servo/servo/pulls |
+    jq '.review_comments_url' |
```

Comment:
We will have to ensure that jq is available on the build machines: https://github.com/servo/saltfs/blob/adf6014621553023cf86382c230cac360bf9f085/upstream-wpt-webhook/init.sls#L26

Reply:
So I think I need to check if `jq` exists first 👀? Or, you mean I can try another way (ex. `grep`) to find the url?

Reply:
We need to make a pull request to saltfs to install the jq package, and then deploy that change to our build machines. Then we can merge this change.

Reply:
Ah, understood! Will send a PR to `saltfs`!

Reply:
Sent at servo/saltfs#954 !